docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -172,17 +172,24 @@ If your {kib} Space doesn't include the Search solution you will have to create
 For example, if you create a {ref}/es-connectors-github.html[GitHub connector] you have to set a `name`, attach it to a new or existing `index`, add your `personal access token` and include the `list of repositories` to synchronize.
 +
 Learn more about configuring and {ref}/es-connectors-usage.html[using connectors] in the Elasticsearch documentation.
-+
-. Create a pipeline and process the data with ELSER.
-+
-To create the embeddings needed by the AI Assistant (weights and tokens into a sparse vector field), you have to create an *ML Inference Pipeline*:
-+
-.. Open the previously created connector and select the *Pipelines* tab.
-.. Select *Copy and customize* button at the `Unlock your custom pipelines` box.
-.. Select *Add Inference Pipeline* button at the `Machine Learning Inference Pipelines` box.
-.. Select *ELSER (Elastic Learned Sparse EncodeR)* ML model to add the necessary embeddings to the data.
-.. Select the fields that need to be evaluated as part of the inference pipeline.
-.. Test and save the inference pipeline and the overall pipeline.
+
+After creating your connector, create the embeddings needed by the AI Assistant. You can do this using either <<obs-ai-search-connectors-ml-embeddings, a machine learning (ML) pipeline>>, which requires the ELSER model, or <<obs-ai-search-connectors-semantic-text, a `semantic_text` field type>>, which can use any available model (ELSER, E5, or a custom model).
```

Comment:
You _could_ format the two options as an unordered list to make it more scannable, but it's up to you!